.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
This will not working AFAIK because you install vendor from source.

Better solution to propose:

* Create a Sonata GitHub token for Travis
* Set up a `GITHUB_OAUTH_TOKEN` private env on each Travis project containing the Github key
* Update the Travis config to install form dist and cache the home composer vendor cache instead like this: https://github.com/Soullivaneuh/IsoCodesValidator/blob/32fb360ec390df114536f68c4760231e6d8825f9/.travis.yml#L37-L46

And then, you will have a powerful quick Travis! :-)

What do you think? I think I need you for the GitHub token setting.